src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/leftcurly/InputLeftCurlyCommentBeforeLeftCurly2.java

```diff
@@ -25,29 +30,29 @@ public class InputLeftCurlyCommentBeforeLeftCurly2 {
     private long[] countList;
 
     void method1() /* comment
-    comment */ { // ok until #11410
+    comment */ { // ok
     }
 
     void method2() /********
-    comment */ {  // ok
+    comment */ { // ok
         if (!Arrays.equals(this.countList, countList)
-            /* || !Arrays.equals(this.whereCreated, other.whereCreated) */ ) { // ok
+            /* || !Arrays.equals(this.whereCreated, other.whereCreated) */ ) {
         }
     }
 
     void method3() /*****
-    ********/ { } // ok until #11410
+    ********/ { } // ok
 
 
-    InputLeftCurlyCommentBeforeLeftCurly2() /**************/ { } // ok
+    InputLeftCurlyCommentBeforeLeftCurly2() /**************/ { }
 
     InputLeftCurlyCommentBeforeLeftCurly2(int data) /*
-     ****** comment *********/ { } // ok until #11410
+     ****** comment *********/ { } // ok
 }
 
 class Class {
     private
-    class Node /************/{
+    class Node /************/{ // ok
```

Comment:
please remove "ok", no violation kind of mean "// ok". Lets not change this line at all.

Reply:
done